static/js/message_list_view.js

```diff
@@ -218,6 +218,11 @@ MessageListView.prototype = {
             message_container.msg.unread = unread.message_unread(message_container.msg);
             self._maybe_format_me_message(message_container);
 
+            message_container.is_bot = (function is_bot() {
+                var email = message_container.msg.sender_email;
+                return people.get_by_email(email).is_bot;
+            }());
```

Comment:
I would probably move this up to be next to the `sender` logic; and also the variable should be called `sender_is_bot`, since it's the sender, not the message, that's a bot.